scripts/verify-phase-51-3-pilot-beta-rc-ga-gate-contract.sh

```
@@ -0,0 +1,140 @@
+}
+
+for line in "${forbidden_lines[@]}"; do
+  if contains_forbidden_outside_forbidden_section "${line}"; then
+    echo "Forbidden Phase 51.3 gate contract claim: ${line}" >&2
+    exit 1
+  fi
+done
+
+mac_user_home="$(printf '/%s/' 'Users')"
+unix_user_home="$(printf '/%s/' 'home')"
+windows_user_profile="$(printf '[A-Za-z]:\\\\%s\\\\' 'Users')"
```

Shellcheck (0.11.0)

[info] Want to escape a single quote? echo 'This is how it'\''s done'.

(SC1003)